packages/features/sales-escrows/pages/NFTPage/index.tsx

```diff
@@ -337,13 +342,20 @@ export const NFTPage = () => {
                             {currentOpenAuction?.sale_type?.auction?.config?.auction?.buy_now
                               ?.length > 0 &&
                               principal != verifyOwner && (
+                              BigInt(Number(currentOpenAuction?.sale_type?.auction?.end_date)) >
```

Comment:
If the inner expression returns undefined, the outer expression will be BigInt(Number(undefined)).
That will cause an error: "The number NaN cannot be converted to a BigInt because it is not an integer"